src/main/java/jp/co/soramitsu/sora/crypto/common/Consts.java

```diff
@@ -1,14 +1,16 @@
 package jp.co.soramitsu.sora.crypto.common;
 
 
-public class Consts {
+import lombok.experimental.FieldDefaults;
+import lombok.experimental.UtilityClass;
 
-  public static final String PROOF_KEY = "proof";
-  public static final String ED25519_SHA3_SIGNATURE = "Ed25519Sha3Signature";
-  public static final String ED25519_SHA3_VERIFICATION_KEY = "Ed25519Sha3VerificationKey";
-  public static final String SHA3_256 = "SHA3-256";
-  public static final String SHA3_512 = "SHA3-512";
+@UtilityClass
+@FieldDefaults(makeFinal = true)
+public class Consts {
 
-  private Consts() {
-  }
+  public static String PROOF_KEY = "proof";
```

Comment:
key word static is redundant in this situation, because @UtilityClass marked all members of class as static